spec/server.spec.js

```diff
@@ -23,7 +23,7 @@ const { Message, Headers } = require('@projectriff/message');
 const request = require('superagent');
 
 const HOST = process.env.HOST || '127.0.0.1';
-const HTTP_PORT = 8080;
+const PORT = 8080;
```

Comment:
Since we are no longer passing PORT to the child process explicitly, we should capture a custom value if set.

```suggestion
const PORT = process.PORT || '8080';
```